<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Email</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:rgb(54,54,54);
color:white;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton{
background-color: rgb(0, 170, 127);
border-radius:3px;
}
QPushButton:hover{
border:2px solid white;
}
QPushButton:pressed{
border:3px solid grey;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Your email address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLE">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>example@email.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Subject of the email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="subjectLE">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>35</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Subject tittle of the email</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="placeholderText">
      <string>Type the issue in detail</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Send Mail</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
